editor/osm_auth.cpp

```diff
@@ -280,6 +273,18 @@ string OsmOAuth::FetchRequestToken(SessionID const & sid) const
   }
 }
 
+string OsmOAuth::BuildOAuth2Url() const
+{
+   string requestTokenUrl = m_baseUrl + "/oauth2/authorize";
```

Comment:
```suggestion
   auto const requestTokenUrl = m_baseUrl + "/oauth2/authorize";
```

Reply:
This variable couldn't be `const` because `.append("...")` method fails.

Reply:
My bad, sorry.